packages/blade/src/components/ChatMessage/docs/ChatMessage.stories.tsx

```diff
@@ -0,0 +1,158 @@
+  return (
+    <Box display="flex" flexDirection="column" gap="spacing.5">
+      {messageTypes.map((message, index) => (
+        <ChatMessage senderType="self" messageType={message} key={index}>
+          Hi, Can you help me with the docs?
+        </ChatMessage>
+      ))}
+    </Box>
+  );
+};
+
+export const MessageTypes = ChatMessageMessageTypeTemplates.bind({});
+MessageTypes.storyName = 'Message Types';
+MessageTypes.args = {};
+
+const ChatMessageSenderTypeTemplates: StoryFn<typeof ChatMessage> = () => {
+  return (
+    <Box display="flex" flexDirection="column" gap="spacing.5">
+      <ChatMessage senderType="self" messageType="default">
+        Hi, Can you help me with the docs?
+      </ChatMessage>
+      <ChatMessage
+        senderType="other"
+        leading={<RayIcon size="xlarge" color="surface.icon.onSea.onSubtle" />}
+      >
+        Hi, Can you help me with the docs?
+      </ChatMessage>
+    </Box>
+  );
+};
+
+export const SenderTypes = ChatMessageSenderTypeTemplates.bind({});
+SenderTypes.storyName = 'Sender Types';
+
+const ChatMessageLoadingTemplates: StoryFn<typeof ChatMessage> = () => {
+  return (
+    <Box display="flex" flexDirection="column" gap="spacing.5">
+      <ChatMessage
+        isLoading
+        senderType="other"
+        loadingText="Analyzing your response..."
+        leading={<RayIcon size="xlarge" color="surface.icon.onSea.onSubtle" />}
+      />
+    </Box>
+  );
+};
+
+export const Loading = ChatMessageLoadingTemplates.bind({});
+Loading.storyName = 'Loading Chat Message';
+
+const ChatMessageErrorTemplates: StoryFn<typeof ChatMessage> = () => {
+  return (
+    <Box display="flex" flexDirection="column" gap="spacing.5">
+      <ChatMessage
+        validationState="error"
+        senderType="self"
+        messageType="last"
+        errorText="Message not sent. Tap to retry."
+      >
+        Can you help me with the docs?
+      </ChatMessage>
+    </Box>
+  );
+};
+
+export const Error = ChatMessageErrorTemplates.bind({});
+Error.storyName = 'Error Chat Message';
+
+const ChatMessageBodyTemplates: StoryFn<typeof ChatMessage> = () => {
+  return (
+    <Box>
+      <ChatMessage
+        senderType="other"
+        leading={<RayIcon size="xlarge" color="surface.icon.onSea.onSubtle" />}
+      >
+        <Card>
+          <CardBody>
+            <Box display="flex" gap="8px" flexDirection="column">
+              <Text variant="body" size="medium">
+                Where do you want to collect payments?
+              </Text>
+              <RadioGroup>
+                <Radio value="website">Website</Radio>
+                <Radio value="android">Android App</Radio>
+                <Radio value="ios">iOS App</Radio>
+              </RadioGroup>
+            </Box>
+          </CardBody>
+        </Card>
+      </ChatMessage>
+    </Box>
+  );
+};
+
+export const ChatMessageBody = ChatMessageBodyTemplates.bind({});
+ChatMessageBody.storyName = 'Chat Message Body';
```

Comment:
Can you add a story for `footerActions` Prop as well?

Reply:
done